scripts/cluster_network.py

```diff
@@ -557,12 +557,12 @@ def plot_busmap_for_n_clusters(n, n_clusters, solver_name="scip", fn=None):
     ):  # also available: linemap_positive, linemap_negative
         getattr(clustering, attr).to_csv(snakemake.output[attr])
 
-    nc.shapes = n.shapes.copy()
+    # nc.shapes = n.shapes.copy()
     for which in ["regions_onshore", "regions_offshore"]:
         regions = gpd.read_file(snakemake.input[which])
         clustered_regions = cluster_regions((clustering.busmap,), regions)
         clustered_regions.to_file(snakemake.output[which])
-        append_bus_shapes(nc, clustered_regions, type=which.split("_")[1])
```

Comment:
was the memory issue even coming up for the clustered regions?

Reply:
In principle yes, if you have a lot of clusters like 1000.